frontend/src/utils/tokens.ts

```diff
@@ -1,13 +1,217 @@
 import BigNumber from 'bignumber.js'
 
-import { NTRN_DENOM, NTRN_CONFIG } from '@config/environment'
-import { formatToSignificantDigits, unitsToValue, valueToUnits } from './number'
+import defaultLogo from '@assets/tokens/alt.png'
+import ntrnLogo from '@assets/tokens/ntrn-logo.svg'
+import atomLogo from '@assets/tokens/atom-logo.svg'
+import datomLogo from '@assets/tokens/datom-logo.svg'
+import statomLogo from '@assets/tokens/statom-logo.svg'
+import stosmoLogo from '@assets/tokens/stosmo-logo.svg'
+import stdydxLogo from '@assets/tokens/stdydx-logo.svg'
+import sttiaLogo from '@assets/tokens/sttia-logo.svg'
+import stdymLogo from '@assets/tokens/stdym-logo.svg'
+import ethLogo from '@assets/tokens/eth-logo.svg'
+import btcLogo from '@assets/tokens/btc-logo.svg'
+import axlLogo from '@assets/tokens/axl-logo.svg'
+import osmoLogo from '@assets/tokens/osmo-logo.svg'
+import seiLogo from '@assets/tokens/sei-logo.svg'
+import aktLogo from '@assets/tokens/akt-logo.svg'
+import injLogo from '@assets/tokens/inj-logo.svg'
+import tiaLogo from '@assets/tokens/tia-logo.svg'
+import stkatomLogo from '@assets/tokens/stkatom-logo.svg'
+import dymLogo from '@assets/tokens/dym-logo.svg'
+import usdcLogo from '@assets/tokens/usdc-logo.svg'
+
+import { formatToSignificantDigits, unitsToValue, valueToUnits } from '@utils/number'
 
-BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_DOWN })
 export const SIGNIFICANT_DIGITS = 5
 
+export interface TokenConfig {
+  icon: string,
+  symbol: string,
+  exponent: number,
+  pythId: string | undefined,
+}
+
 export type Denom = string
 
+export const tokenConfigs: Map<Denom, TokenConfig> = new Map([
+  [
+    "untrn",
+    {
+      icon: ntrnLogo,
+      symbol: "NTRN",
+      exponent: 6,
+      pythId: "a8e6517966a52cb1df864b2764f3629fde3f21d2b640b5c572fcd654cbccd65e",
+    },
+  ],
+  [
+    "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2",
+    {
+      icon: atomLogo,
+      symbol: "ATOM",
+      exponent: 6,
+      pythId: "b00b60f88b03a6a625a8d1c048c3f66653edf217439983d037e7222c4e612819",
+    },
+  ],
+  [
+    "ibc/EA6E1E8BA2EB9F681C4BD12C8C81A46530A62934F2BD561B120A00F46946CE87	",
```

Comment:
Will the extra spaces at the end here cause trouble?

Reply:
Must've been my cat walking on the keyboard :laughing: